src/compressor/QatAccel.cc

```diff
@@ -14,6 +14,8 @@
 #include <qatzip.h>
 #include "QatAccel.h"
 
+#define SESSIONS_NUM 200
```

Comment:
might want to fix the typo of "crush" in the commit message.

Reply:
better off add a comment explaining why we use 200 here.

Reply:
```suggestion
static const unsigned SESSIONS_NUM = 200;
```

Reply:
200 is a test parameter, so I make it configure. And I set default value is 200.

Change "crush" to "crash".